src/methods/limitOrders/helpers/types.ts

```diff
@@ -28,6 +28,7 @@ export type LimitOrderState =
   | 'PENDING'
   | 'FULFILLED'
   | 'CANCELLED'
+  | 'SUSPENDED'
```

Comment:
Can you add a comment what SUSPENDED means :pray:

Reply:
@Velenir added